charts/renovate/values.yaml

```
@@ -10,6 +10,8 @@ fullnameOverride: ''
 cronjob:
   # -- Schedules the job to run using cron notation
   schedule: '0 1 * * *'  # At 01:00 every day
+  # -- You can specify a time zone for a CronJob by setting timeZone to the name of a valid time zone. (starting with k8s 1.27) https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/#time-zones
+  timeZone: '0 1 * * *'  # At 01:00 every day
```

no, this will cause errors on older kubernetes 
```suggestion
  timeZone: ''
```

---

copy/paste error, sorry.

I corrected it